Fix Templates; update dependencies (#428)
carpentries/sandpaper: inst/templates/contributing-template.txt

1. If you do not have a [GitHub][github] account, you can [send us comments by
   email][contact]. However, we will be able to respond more quickly if you use
   one of the other methods described below.

2. If you have a [GitHub][github] account, or are willing to [create
   one][github-join], but do not know how to use Git, you can report problems
   or suggest improvements by [creating an issue][issues]. This allows us to
   assign the item to someone and to respond to it in a threaded discussion.

3. If you are comfortable with Git, and would like to add or change material,
   you can submit a pull request (PR). Instructions for doing this are
   [included below](#using-github).

Note: if you want to build the website locally, please refer to [The Workbench
documentation][template-doc].

### Where to Contribute

1. If you wish to change this lesson, add issues and PR here.
2. If you wish to change the template used for workshop websites, please refer
   to [The Workbench documentation][template-doc].

### What to Contribute

There are many ways to contribute, from writing new exercises and improving
existing ones to updating or filling in the documentation and submitting [bug

> reports][issues] about things that don't work, aren't clear, or are missing. If

Reviewer: ```suggestion
reports][issues] about things that do not work, are not clear, or are missing. If
```

Reviewer: I believe it is more accessible to expand negative contractions such as "don't", "can't", etc.

Author: indeed it is! Thank you!

you are looking for ideas, please see [the list of issues for this

> repository][issues], or the issues for [Data Carpentry][dc-issues] and
> [Software Carpentry][swc-issues] projects.

Reviewer: ```suggestion
repository][issues], or the issues for [Data Carpentry][dc-issues], [Library Carpentry][lc-issues], and
[Software Carpentry][swc-issues] projects.
```

Each lesson has two maintainers who review issues and pull requests or
encourage others to do so. The maintainers are community volunteers, and have
final say over what gets merged into the lesson.

### Other Resources

The Carpentries is a global organisation with volunteers and learners all over
the world. We share values of inclusivity and a passion for sharing knowledge,
teaching and learning. There are several ways to connect with The Carpentries
community listed at <https://carpentries.org/connect/> including via social
media, slack, newsletters, and email lists. You can also [reach us by
email][contact].

[contact]: mailto:team@carpentries.org
[cp-site]: https://carpentries.org/
[dc-issues]: https://github.com/issues?q=user%3Adatacarpentry
[dc-lessons]: https://datacarpentry.org/lessons/
[dc-site]: https://datacarpentry.org/
[discuss-list]: https://lists.software-carpentry.org/listinfo/discuss
[github]: https://github.com
[github-flow]: https://guides.github.com/introduction/flow/
[github-join]: https://github.com/join
[how-contribute]: https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github
[issues]: https://carpentries.org/help-wanted-issues/

> [repo]: https://github.com/swcarpentry/shell-novice/

Reviewer: Is there a way we can avoid using swcarpentry/shell-novice (or any other specific repository) here?

Author: oof yeah, that is ew

Author: It's one of those things that really should be automatic, but it's ultimately a "fixme" kind of situation.

Reviewer: Can we add something to the template README, instructing people to fix this as part of their initial setup?

Author: Yes we can. I'm going to make it an `https://example.com/FIXME` link so people can search for FIXME, but yes, it's something that we can instruct people to fix.
